### PR TITLE
Reintroduce the uhfqa.qa_delay function as uhfqa.qas.adjusted_delay

### PR DESCRIPTION
**Pull request by ghost**
With the refactoring the function `qa_delay` was removed from the UHFQA driver. 

This function is not really part of the sequencer generator, but mostly a patch of the behavior of the ``/DEV…​./QAS/n/DELAY`` node to make it more reasonable. (ideally that would be part of the firmware, but that is not possible because of multiple reasons.

Since the ``delay`` node is part of the ``qas`` node we should add it there.

### CODE REVIEW

**Comment by tobiasah** [comment]
@YakBizzarro I still need to write tests for it but wanted to get you in feedback first

**Comment by YakBizzarro** [comment]
LGTM, thanks!

File: src/zhinst/toolkit/driver/devices/uhfqa.py [hunk 1]
```diff
@@ -66,6 +66,48 @@ def crosstalk_matrix(self, matrix: Numpy2DArray = None) -> t.Optional[Numpy2DArr
                 for c in range(cols):
                     self.crosstalk.rows[r].cols[c](matrix[r, c])
 
+    def adjusted_delay(self, value: int = None) -> int:
```
Comment (markheik) [inline src/zhinst/toolkit/driver/devices/uhfqa.py:69]: value is Optional

**Review by markheik — COMMENTED** [review]
Fix typing and then LGTM

**Review by markheik — APPROVED** [review]

File: src/zhinst/toolkit/driver/devices/uhfqa.py (hunk 1 above)
Comment (tobiasah) [reply src/zhinst/toolkit/driver/devices/uhfqa.py:69]: typing is ok. 😁

**Pull request merged**